SofaKernel/framework/sofa/defaulttype/DataTypeInfo.h

```diff
@@ -1284,6 +1294,33 @@ struct DataTypeInfo< sofa::helper::vector<bool,Alloc> > : public VectorTypeInfo<
     static void* getValuePtr(sofa::helper::vector<bool,Alloc>& /*data*/) { return NULL; }
 };
 
+template<class Alloc>
+struct DataTypeInfo< sofa::helper::vector<std::string,Alloc> > : public VectorTypeInfo<sofa::helper::vector<std::string,Alloc> >
```

Comment:
your PR says "& improved doc" but I think you forget some while adding your new struct ...